datasets/medmcqa/README.md

```diff
@@ -0,0 +1,231 @@
+
+MedMCQA provides an open-source dataset for the Natural Language Processing community.
+It is expected that this dataset would facilitate future research toward achieving better QA systems.
+The dataset contains questions about the following topics:
+
+- Anesthesia
+- Anatomy
+- Biochemistry
+- Dental
+- ENT
+- Forensic Medicine (FM)
+- Obstetrics and Gynecology (O&G)
+- Medicine
+- Microbiology
+- Ophthalmology
+- Orthopedics
+- Pathology
+- Pediatrics
+- Pharmacology
+- Physiology
+- Psychiatry
+- Radiology
+- Skin
+- Preventive & Social Medicine (PSM)
+- Surgery
+
+### Supported Tasks and Leaderboards
+
+multiple-choice-QA, open-domain-QA: The dataset can be used to train a model for multi-choice questions answering, open domain questions answering. Questions in these exams are challenging and generally require deeper domain and language understanding as it tests the 10+ reasoning abilities across a wide range of medical subjects & topics.
+
+### Languages
+
+The questions and answers are available in English.
+
+## Dataset Structure
+
+### Data Instances
+
+```
+{
+    "question":"A 40-year-old man presents with 5 days of productive cough and fever. Pseudomonas aeruginosa is isolated from a pulmonary abscess. CBC shows an acute effect characterized by marked leukocytosis (50,000 mL) and the differential count reveals a shift to left in granulocytes. Which of the following terms best describes these hematologic findings?",
+    "exp": "Circulating levels of leukocytes and their precursors may occasionally reach very high levels (>50,000 WBC mL). These extreme elevations are sometimes called leukemoid reactions because they are similar to the white cell counts observed in leukemia, from which they must be distinguished. The leukocytosis occurs initially because of the accelerated release of granulocytes from the bone marrow (caused by cytokines, including TNF and IL-1) There is a rise in the number of both mature and immature neutrophils in the blood, referred to as a shift to the left. In contrast to bacterial infections, viral infections (including infectious mononucleosis) are characterized by lymphocytosis Parasitic infestations and certain allergic reactions cause eosinophilia, an increase in the number of circulating eosinophils. Leukopenia is defined as an absolute decrease in the circulating WBC count.",
+    "cop":1,
+    "opa":"Leukemoid reaction",
+    "opb":"Leukopenia",
+    "opc":"Myeloid metaplasia",
+    "opd":"Neutrophilia",
+    "subject_name":"Pathology",
+    "topic_name":"Basic Concepts and Vascular changes of Acute Inflammation",
+    "id":"4e1715fe-0bc3-494e-b6eb-2d4617245aef",
+    "choice_type":"single"
+}
+```
+### Data Fields
+
+- `id` : a string question identifier for each example
+- `question` : question text (a string)
+- `opa` : Option A
+- `opb` : Option B
+- `opc` : Option C
+- `opd` : Option D
+- `cop` : Correct option, i.e., 1,2,3,4
+- `choice_type` : Question is `single-choice` or `multi-choice`
```

Comment:
The values appearing for `choice_type` are `single` and `multi` (instead of `single-choice` or `multi-choice`).

On the other hand I do not understand the meaning of this parameter. I would naively think that in multi choice questions, more than a single answer can be correct; however `cop` always contain one single correct answer.

What is the difference between single and multi?

Reply:
Sorry for the confusion, let me explain by example : 

**Multichoice** 

Here option is one but each option contains a combination of different suboptions.

```
{'choice_type': 'multi',
   'cop': 0,
   'exp': 'For intubation: 1st step: Morning sniffing position: extension of atlantooccipital joint and flexion of the neck. 2nd step: Introduction of laryngoscope 3rd step: Push the tongue away, visualize the epiglottis, and aryepiglottic fold, hinge the tip of the laryngoscope with the aryepiglottic fold, and lift the laryngoscope to visualize the glottic opening. 4th step: Introduce ETT and inflate the cuff 5th check for bilateral breath sound FIXATION of the ETT.',
   'id': 'fb79561d-8beb-4a31-aaa5-9350e20b0caa',
   'opa': 'ABCDE',
   'opb': 'DBCEA',
   'opc': 'ACBED',
   'opd': 'CBAED',
   'question': 'Steps of intubation - arrange in sequence:- a. Head extension and flexion of neck b. Introduction of laryngoscope c. Inflation of cuff d. Check breath sounds with a stethoscope e. fixation of the tube to prevent dislodgement',
   'subject_name': 'Anaesthesia',
   'topic_name': 'AIIMS 2019'}
 ```


**Single choice**

Here the option is single

```{'choice_type': 'single',
   'cop': 0,
   'id': '2c98bee7-541d-4bef-a8f8-49ffdb6e2213',
   'opa': '4,000-11,000',
   'opb': '40000 -80000',
   'opc': '13000-17000',
   'opd': '1000-3000',
   'question': 'Normal WBC count',
   'subject_name': 'Dental'}```